src/test/java/org/cbioportal/persistence/util/fakeclient/MockInMemoryRedissonClient.java

```diff
@@ -231,6 +231,7 @@ public RLock getMultiLock(RLock... rLocks) {
      * @deprecated
      */
     @Override
+    @Deprecated
```

Comment:
Please write an issue and leave a calendar reminder to fully remove this in the future.

Reply:
They're required by the implementation of RedissonClient and still marked `@Deprecated` in the latest version [(v3.27.2)](https://www.javadoc.io/doc/org.redisson/redisson/latest/deprecated-list.html).